### PR TITLE
fix: where to put links confusion

### PR DESCRIPTION
**Pull request by mohit1607**
The json object for only links is updated to the whole json of a profile, so that no furher confusion about the structure of json happen.

## Fixes Issue
Quckstart improvement - explain json structure #2538

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot (5732)
## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2593"></a>

### CODE REVIEW

File: pages/docs/quickstart.mdx [hunk 1]
```diff
@@ -54,13 +54,20 @@ To do this, you need a GitHub account. If you do not have one yet, you can creat
...
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
+  "avatar": "https://github.com/SaraJaoude.png",
   "links": [
     {
       "name": "Follow me on Twitter",
       "url": "https://twitter.com/SaraJaoude",
       "icon": "FaTwitter"
-    }
+    },
```
Comment (eddiejaoude) [inline pages/docs/quickstart.mdx:68]: I think this trailing comma makes the json invalid?

Comment (Panquesito7) [reply pages/docs/quickstart.mdx:68]: Yep, it makes it invalid, unless there's another field next to it.

Comment (mohit1607) [reply pages/docs/quickstart.mdx:68]: ok I will remove this

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🚀

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**